benchmark

```diff
@@ -299,6 +297,10 @@ def main():
                         default=1,
                         help='Number of runs for each compilation')
 
+    parser.add_argument('--path', dest='path', nargs='+',
+                        default=["/usr/local/bin"],
```

Comment:
I suggested `"/usr/local/bin"` here, but I'm open for other default option!

Reply:
could actually be defaulted to $PATH?

Reply:
Makes actually more sense to me so commited a patch.

Reply:
is the current behavior kept if the --path is not given?